api/src/main/java/io/lumeer/api/dto/JsonCollection.java

```diff
@@ -102,8 +104,8 @@ public void updateAttribute(final String attributeId, final Attribute attribute)
    }
 
    @Override
-   public void deleteAttribute(final String attributeName) {
-      attributes.removeIf(attribute -> AttributeUtil.isEqualOrChild(attribute, attributeName));
+   public void deleteAttribute(final String attributeId) {
+      attributes.removeIf(attribute -> attribute.getId().equals(attributeId));
```

Comment:
This change breaks deleting child attributes.

Reply:
Really? How do we know which are child attributes?

Reply:
Those which names start with the name of the deleted attribute + `.`. It was easier before. Now we first need to get the deleted attribute and then remove its children.

Reply:
Oh sorry, I misunderstood how we will use `name` field. I will definetely revert this change.